### DISCUSSION TITLE
How to edit/delete boot animation?

### DISCUSSION THREAD

**Discussion by HoneyBear4Lyfe** [seerge/g-helper#3518] [Q&A] [answered]
I've been using Seerge's excellent replacement for about a year now, so Armoury Crate is just an unhappy memory. The one thing that I miss about it is the ability to change my boot video. I am certainly not suggesting that such a feature be added to g-helper, but I'm wondering if anybody can maybe direct me to where I might be able to manually change it? I'd Like to simply delete it since it often makes an unwanted but unavoidable loud noise while my partner is asleep.

**Comment by HoneyBear4Lyfe** [comment]
I assume it's saved in the same place for all ROG computers, but in case it's not, I have a Zephyrus G16

**Comment by seerge** [comment]
@HoneyBear4Lyfe hello,

Most likely it's stored on UEFI (SYSTEM) partition. And honestly I **strongly advice against** editing something there, as you may end up with not working device.

You can read on how to [mount it here for example](https://askubuntu.com/questions/429610/uninstall-grub-and-use-windows-bootloader) (all steps except last ,as it's not relevant)

**Reply by HoneyBear4Lyfe** [reply]
hmmm, seems like a risky move, I guess I probably shouldn't do it. I know just enough to likely break something.
Do you think I could get away with downloading armoury crate, deleting the video, then deleting armoury crate?

**Reply by seerge** [reply, marked as answer]
@HoneyBear4Lyfe Armoury Crate can't edit that animation either.

**Reply by HoneyBear4Lyfe** [reply]
shoot, I guess I'm out of luck then. Thanks for your help, as always!
